sdk/nodejs/storage/zMixins.ts

```diff
@@ -228,16 +228,18 @@ export class BlobEventSubscription extends appservice.EventSubscription<BlobCont
             connection: bindingConnectionKey,
         }];
 
+        args.bindings = appservice.mergeBindings(bindings, args.bindings);
+
         // Place the mapping from the well known key name to the storage account connection string in
         // the 'app settings' object.
         const account = pulumi.all([resourceGroupName, container.storageAccountName])
-                              .apply(([resourceGroupName, storageAccountName]) =>
-                                storage.getAccount({ resourceGroupName, name: storageAccountName }));
+                            .apply(([resourceGroupName, storageAccountName]) =>
+                              storage.getAccount({ resourceGroupName, name: storageAccountName }));

```

Comment:
can you undo the indentation change.  it was intentional to align the .all and .apply.  thanks!

Reply:
fixed in d783c138f08bff57325be4e7fb98e3a6933d5b8b